apps/vs-code-designer/src/app/commands/createNewCodeProject/createNewCodeProject.ts

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { funcVersionSetting, projectLanguageSetting, projectOpenBehaviorSetting, projectTemplateKeySetting } from '../../../constants';
+import { localize } from '../../../localize';
+import { addLocalFuncTelemetry, tryGetLocalFuncVersion, tryParseFuncVersion } from '../../utils/funcCoreTools/funcVersion';
+import { getGlobalSetting, getWorkspaceSetting } from '../../utils/vsCodeConfig/settings';
+import { FolderListStep } from '../createNewProject/createProjectSteps/FolderListStep';
+import { OpenBehaviorStepCodeProject } from './CodeProjectBase/OpenBehaviorStepCodeProject';
+import { OpenFolderStepCodeProject } from './CodeProjectBase/OpenFolderStepCodeProject';
+import { NewCodeProjectTypeStep } from './createCodeProjectSteps/NewCodeProjectTypeStep';
+import { setWorkspaceName } from './createCodeProjectSteps/SetWorkspaceName';
+import { setMethodName } from './createCodeProjectSteps/createFunction/setMethodName';
+import { setNameSpace } from './createCodeProjectSteps/createFunction/setNamepSpace';
+import { isString } from '@microsoft/utils-logic-apps';
+import { AzureWizard } from '@microsoft/vscode-azext-utils';
+import type { IActionContext } from '@microsoft/vscode-azext-utils';
+import { latestGAVersion, OpenBehavior } from '@microsoft/vscode-extension';
+import type { ICreateFunctionOptions, IFunctionWizardContext, ProjectLanguage, ProjectVersion } from '@microsoft/vscode-extension';
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import { window } from 'vscode';
+
+export async function createNewCodeProjectFromCommand(
+  context: IActionContext,
+  folderPath?: string | undefined,
+  language?: ProjectLanguage,
+  version?: ProjectVersion,
+  openFolder = true,
+  templateId?: string,
+  functionName?: string,
+  functionSettings?: { [key: string]: string | undefined }
+): Promise<void> {
+  await createNewCodeProjectInternal(context, {
```

Comment:
NIT: I feel that you don't need to follow this from the other NewCodeProject command, by directly having the call to the other function or having all the logic from the other function in this should work fine